lib/pages/edit_task.dart

```diff
@@ -3,65 +3,55 @@ import 'package:go_router/go_router.dart';
 import 'package:task_list_app/constants/constants.dart';
 import 'package:task_list_app/models/task.dart';
 
-class EditTask extends StatefulWidget {
+class EditTask extends StatelessWidget {
   final Task task;
   final Function(Task) onTaskUpdated;
-  const EditTask({super.key, required this.task, required this.onTaskUpdated});
-
-  @override
-  State<EditTask> createState() => _EditTaskState();
-}
 
-class _EditTaskState extends State<EditTask> {
-  late TextEditingController _controller;
-  @override
-  void initState() {
-    super.initState();
-    _controller = TextEditingController(text: widget.task.name);
-  }
+  const EditTask({super.key, required this.task, required this.onTaskUpdated});
 
   @override
   Widget build(BuildContext context) {
+    TextEditingController controller = TextEditingController(text: task.name);
+
     return Form(
-        key: const Key('Form'),
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.start,
-          children: [
-            Padding(
-              padding: const EdgeInsets.fromLTRB(0, 0, 0, 42),
-              child: TextField(
-                style: TextStyle(color: iconColor),
-                controller: _controller,
-                decoration: InputDecoration(
-                  labelText: 'Edit Task Name',
-                  labelStyle: TextStyle(color: white, fontSize: 20),
-                ),
+      key: const Key('Form'),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: [
+          Padding(
+            padding: const EdgeInsets.fromLTRB(0, 0, 0, 42),
```

Comment:
Please change this to:

```suggestion
            padding: const EdgeInsets.only(bottom: 42),
```

Reply:
done